README.md

```diff
@@ -109,3 +71,45 @@ The following files will be created:
 - `index.html`: start page for the project
 - `pyscript.toml`: project metadata and config file
 - `main.py`: a "Hello world" python starter module
+
+#### Use --wrap to embed a python file OR a command string
+
+- ##### Embed a Python script into a PyScript HTML file
+
+```shell
+$ pyscript create --wrap <filename.py>
+```
+
+This will generate a project i.e. a new directory named `filename` under the current directory.
+
+Similar to the above, interactive prompts will further ask for metadata information.
+
+The following files will be created:
+- `filename.html`: embeds the code of `filename.py` in a `<py-script>` tag.
```

Comment:
-0.5 on this one. This makes the project layout a bit less predictable which is less optimal to the user.

I'd prefer to have the project creation output to be standard and predicable: index.html, main.py and pyscript.toml ; no matter what's the name of the input

Reply:
I see, so we don't want the python code to be inline between `<py-script>` tags and want it to be via
`<py-script src=...>` instead?

Reply:
Done!